code/game/objects/items.dm

```diff
@@ -311,7 +311,7 @@ GLOBAL_DATUM_INIT(welding_sparks_prepdoor, /mutable_appearance, mutable_appearan
 		playsound(user, "acid_hit", 25)
 		var/mob/living/carbon/human/H = user
 		H.emote("pain")
-		var/raw_damage = current_acid.acid_damage * 0.25 //It's spread over 4 areas.
+		var/raw_damage = initial(current_acid.acid_damage) * 0.25 //It's spread over 4 areas.
```

Comment:
why initial ? acid damage isn't changing, is it

Reply:
I don't think this changes. Why did we change this?

Reply:
Got a runtime and this fixed it

Reply:
this really shouldn't cause an issue, show me the error you get, something else is going wrong.